src/igvfd/types/file.py

```diff
@@ -290,24 +290,32 @@ def assay_titles(self, request, file_set):
         return list(assay_titles)
 
     @calculated_property(
+        define=True,
         condition='analysis_step_version',
         schema={
-            'title': 'Workflow',
+            'title': 'Workflows',
             'description': 'The workflow used to produce this file.',
-            'type': 'string',
-            'linkTo': 'Workflow',
+            'type': 'array',
+            'minItems': 1,
+            'uniqueItems': True,
+            'items': {
+                'title': 'Workflow',
+                'type': 'string',
```

Comment:
This will also cause the bug which will be resolved by IGVF-2831, in the mean time, we should keep this matching other arrays, like `['string', 'object']`